packages/deploy-utils/src/setup-credentials/index.ts

```diff
@@ -26,60 +28,16 @@ const sharedKeys = {
 const base: Record<TargetEnvironment, Record<string, string | undefined>> = {
```

Comment:
Could we delete the Sentry values above?

Reply:
I don't think so, `add_expansions_to_env` adds the expansions as environment variables but I think they still need to be added to the JSON manually. I tested by deploying to staging via the deploy task, and couldn't receive any reports to Sentry

Reply:
Interesting, I think that has to do with env-cmd overriding the variables set in the environment with the file variables. We could potentially use the `--no-override` [flag](https://www.npmjs.com/package/env-cmd#-help). Any thoughts?